cmd/kubeless/function.go

```diff
@@ -100,14 +102,107 @@ func parseMemory(mem string) (resource.Quantity, error) {
 	return quantity, nil
 }
 
-func getFunctionDescription(funcName, ns, handler, funcContent, deps, runtime, topic, schedule, runtimeImage, mem string, triggerHTTP bool, envs, labels []string, defaultFunction spec.Function) (f *spec.Function, err error) {
+func getFileSha256(file string) (string, error) {
+	var checksum string
+	h := sha256.New()
+	ff, err := os.Open(file)
+	if err != nil {
+		return checksum, err
+	}
+	defer ff.Close()
+	_, err = io.Copy(h, ff)
+	if err != nil {
+		return checksum, err
+	}
+	checksum = hex.EncodeToString(h.Sum(nil))
+	return checksum, err
+}
+
+// Convert MB to Bytes as int64
+func mbtoInt64(mb int) int64 {
+	return int64(mb * 1024 * 1024)
+}
+
+func uploadFunction(file string, cli kubernetes.Interface) (string, string, error) {
+	var checksum string
+	stats, err := os.Stat(file)
+	if stats.Size() > mbtoInt64(50) { // TODO: Make the max file size configurable
```

Comment:
or it can just be a constant number ?

Reply:
yeah, I wrote the function to be a little bit more readable (it is easier to see that the limit is 50 MB if it is (50 * 1024 * 1024 rather than if it is 52428800)